client/src/App.js

```diff
@@ -127,14 +110,75 @@ function LoginForm(props) {
   );
 }
 
-function LogoutBox(props) {
+function MainContent(props) {
   return (
     <Col>
       <p>Welcome, {props.user.username}!</p>
       <p>Role: {props.user.role}</p>
+      {
+        props.user.role === 'ADMIN' ?
+          <NewTaskForm />
+          : undefined
+      }
       <Button onClick={props.doLogOut}>Logout</Button>
     </Col>
   );
 }
 
+function NewTaskForm() {
+  const [name, setName] = useState('');
+  const [time, setTime] = useState('');
+
+  const createNewService = async (event) => {
+    event.preventDefault();
+    const service = {
+      name: name,
+      time: parseFloat(time)
+    };
+
+    let valid = true;
+    if (name === '' || isNaN(parseFloat(time)) || parseFloat(time) === 0.0)
+      valid = false;
+
+    if (valid) {
+      await API.createService(service);
+    } else {
+      console.log('Invalid form');
+    }
+  }
+
+  return (
+    <Form>
+      <Row>
+
+        <Col className='alignCenter'>
+          <Form.Group className="mb-3" controlId="formServiceName">
```

Comment:
Maybe in this case using a dropbox menu it's a little better for user experience.